filter/expand_list_test.go

```diff
@@ -8,8 +8,8 @@ import (
 
 func TestExpandList(t *testing.T) {
 	tests := []struct {
-		name string
-		list string
+		name   string
+		record string // default: ",,"
```

Comment:
Having a default for a test case is slightly confusing, IMO it's better for tests to be explicit, and so in this case setting `record` to `",,"` when this is the behavior of the case we're testing